addons/binding/org.openhab.binding.bigassfan/ESH-INF/thing/thing-types.xml

```diff
@@ -12,12 +12,15 @@
 			<!-- Channels related to fan functions -->
 			<channel id="fan-power" typeId="fan-power" />
 			<channel id="fan-speed" typeId="fan-speed" />
-			<channel id="fan-direction" typeId="fan-direction" />
+			<channel id="fan-direction" typeId="fan-direction"></channel>
 			<channel id="fan-auto" typeId="fan-auto" />
 			<channel id="fan-whoosh" typeId="fan-whoosh" />
 			<channel id="fan-smartmode" typeId="fan-smartmode" />
+			<channel id="fan-learn-speed-min" typeId="fan-learn-speed-min" />
+			<channel id="fan-learn-speed-max" typeId="fan-learn-speed-max" />
 			<channel id="fan-speed-min" typeId="fan-speed-min" />
-			<channel id="fan-speed-max" typeId="fan-speed-min" />
```

Comment:
This looks not backward compatible because fan-speed-min/max suddenly get another meaning, should we publish in the release note to inform users?